doc/modules/impute.rst

```diff
@@ -61,30 +61,39 @@ values than observed values.
 Multivariate feature imputation
 ===============================
 
-A more sophisticated approach is to use the :class:`MICEImputer` class, which
-implements the Multivariate Imputation by Chained Equations technique. MICE
-models each feature with missing values as a function of other features, and
-uses that estimate for imputation. It does so in a round-robin fashion: at
-each step, a feature column is designated as output `y` and the other feature
-columns are treated as inputs `X`. A regressor is fit on `(X, y)` for known `y`.
-Then, the regressor is used to predict the unknown values of `y`. This is
-repeated for each feature, and then is done for a number of imputation rounds.
-Here is an example snippet::
+A more sophisticated approach is to use the :class:`ChainedImputer` class, which
+implements MICE: Multivariate Imputation by Chained Equations. MICE is usually used
+to generate multiple imputations, but :class:`ChainedImputer` generates a single
+(averaged) imputation, which is why it is not named `MICEImputer`. MICE models each
```

Comment:
I think we need to link those terms to a glossary entry that explains:

- univariate vs multivariate imputation
- single vs multiple imputation (increasing the number of samples in the resulting transformed dataset).

Reply:
Good idea. There is no entry for `imputation`. I'll add one and include these points.